metafacture-runner/src/main/dist/examples/transform/pica-to-marcxml/pica-to-marcxml_fix.flux

```diff
@@ -0,0 +1,13 @@
+// opens file 'fileName', interprets the content as non normalized serialized
+// pica+, encode as marc, morphs to marcxml and writes to standard out
```

Comment:
"transforms" instead of "morphs"?

Reply:
Resolved, see: https://github.com/metafacture/metafacture-core/pull/663/files#r1943123101 and https://github.com/metafacture/metafacture-core/pull/663/files#r1943123531